api/resources.py

```diff
@@ -54,22 +54,29 @@ class BotListOrCreateResource(Resource):
     def get(self):
         if request.args.get("search"):
             search_parameter = request.args.get("search")
```

Comment:
```suggestion
        if search_parameter = request.args.get("search"):
```
you can use walrus operator here